### PR TITLE
Prevent caching

### PR DESCRIPTION
**Pull request by darkdrgn2k**

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Refrences #48

File: terraform/ipfs-mirror/nginx-default [hunk 1]
```diff
@@ -42,9 +42,17 @@ server {
     # First attempt to serve request as file, then
     # as directory, then fall back to displaying a 404.
     try_files $uri $uri/ =404;
+    
+    # Prevent Caching
+    expires           0;
+    add_header        Cache-Control private;
```
Comment (benhylau) [inline terraform/ipfs-mirror/nginx-default]: I think `no-cache` is better?

https://www.w3.org/Protocols/HTTP/Issues/cache-private.html

File: terraform/ipfs-mirror/nginx-default [hunk 2]
```diff
@@ -42,9 +42,17 @@ server {
     # First attempt to serve request as file, then
     # as directory, then fall back to displaying a 404.
     try_files $uri $uri/ =404;
+    
+    # Prevent Caching
+    expires           0;
```
Comment (benhylau) [inline terraform/ipfs-mirror/nginx-default]: There are some discussions about `0` as the value, vs. valid date in the past. A little concerned about browser compat.
https://stackoverflow.com/questions/11357430/http-expires-header-values-0-and-1

File: terraform/ipfs-mirror/nginx-default (hunk 1 above)
Comment (benhylau) [reply terraform/ipfs-mirror/nginx-default]: Any value adding `no-store` and `must-revalidate` in addition to this?
https://stackoverflow.com/questions/11357430/http-expires-header-values-0-and-1

File: terraform/ipfs-mirror/nginx-default (hunk 2 above)
Comment (benhylau) [reply terraform/ipfs-mirror/nginx-default]: We will keep this as `0`

**Pull request merged**